x-pack/plugin/autoscaling/src/main/java/org/elasticsearch/xpack/autoscaling/decision/AutoscalingDecision.java

```diff
@@ -16,70 +17,70 @@
 import java.util.Objects;
 
 /**
- * Represents an autoscaling decision.
+ * Represents an autoscaling decision from a single decider
  */
 public class AutoscalingDecision implements ToXContent, Writeable {
 
-    private final String name;
+    private final AutoscalingCapacity requiredCapacity;
+    private final Reason reason;
 
-    public String name() {
-        return name;
+    public interface Reason extends ToXContent, NamedWriteable {
+        String getSummary();
```

Comment:
If these are expected to be immutable (i.e., there's never a setter), we have a rather loose guideline that we'd name it `summary`.

Reply:
:+1: , fixed in 75e410b9fad9c4e7a4b5324ee4cb45731af42a04